sdk/macro/src/lib.rs

```diff
@@ -229,7 +229,6 @@ impl Parse for RespanInput {
 /// its only purpose is to override the span of a token
 /// sequence containing `$crate`. For all other purposes,
 /// a more general proc-macro should be used.
-#[rustversion::since(1.46.0)] // `Span::resolved_at` is stable in 1.46.0 and above
```

Comment:
I think since this crate can be built on lower `rustc` versions than 1.75 (if you're using macros that don't deal with `solana-program` or `solana-sdk`) it might make sense to keep this one, or make the crate's MSRV 1.46.0.

That's a four-year-old version of `rustc` at this point, so people shouldn't have issues with either.

Reply:
Yeah I think that macro's just some old bs anyway? https://github.com/anza-xyz/agave/pull/1667

Reply:
Oh, beautiful! If that lands then this PR is good as-is.

Reply:
That PR has landed and I've rebased this branch